src/index.html

```diff
@@ -4,9 +4,40 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+  <title>capstone</title>
 </head>
 <body>
-  <h1 class='hello'>This should be on the screen</h1>
+
+  <header id="nav">
+    <div class="logo item">
+      <a href="#">ISAI-TOLU</a>
+      <!-- <img src="" alt="logo"> -->
```

Comment:
- [ ] It would be better if you make look this logo more as a logo, Currently looks like a navigation item. kindly add an image or make look more like a logo. 
- [ ] Please delete the commented code. 

😄 💯